packages/plugins/src/SearchIndexPlugin.ts

```diff
@@ -9,18 +9,70 @@ type LeafNode = Node & {
   value: string;
 };
 
+interface IndexItem {
+  title: string;
+  route: string;
+  content?: string[];
+  [key: string]: string | string[];
+}
+
+interface SearchRelevancy {
```

Comment:
Can this type be imported from types/SearchIndex?  They look the same but have a different name...

export type SearchConfig = {
  includeScore?: boolean;
  includeMatches?: boolean;
  maxPatternLength?: number;
  ignoreLocation?: boolean;
  threshold?: number;
  keys?: string[] | { name: string; weight: number }[];
};

Reply:
Good point. Added `SearchConfig` to types package.